packages/deployer/src/utils/files.ts

```diff
@@ -1,34 +1,8 @@
 import * as fs from 'fs';
 import * as dotenv from 'dotenv';
-import { Configuration, Receipt } from '../types';
+import { Receipt } from '../types';
 import * as logger from '../utils/logger';
 
-export function parseConfigFile(configPath: string, nodeVersion: string) {
-  logger.debug('Parsing configuration file');
-  let config: Configuration;
-  try {
-    config = JSON.parse(fs.readFileSync(configPath, 'utf8'));
-  } catch (e) {
-    logger.fail('Failed to parse configuration file');
-    throw e;
-  }
-
-  // A more comprehensive validation should be done beforehand
-  // https://github.com/api3dao/airnode/issues/136
```

Comment:
This issue link is lost.

Reply:
I know, integrating the validator is planned anyway.